### ISSUE TITLE
Provide Option to Turn Off Animations

### ISSUE THREAD

**Issue by varbhat** [mozilla-mobile/fenix#16856]
[comment]: # (Please do your best to search for duplicate issues before filing a new issue so we can keep our issue board clean)
[comment]: # (Every issue should have exactly one feature request described in it. Please do not file feedback list tickets as it is difficult to parse them and address their individual points)
[comment]: # (Feature Requests are better when they’re open-ended instead of demanding a specific solution e.g: “I want an easier way to do X” instead of “add Y”)
[comment]: # (Read https://github.com/mozilla-mobile/fenix#i-want-to-file-an-issue for more information)

### What is the user problem or growth opportunity you want to see solved?
Firefox Fenix has too many animations in its user interface. I want an option where i can turn off animation. Also, another option where we can set amount of animation ( like 0.1x , 0.2x). Also, another option when set,respects animation setting of the device.

### How do you know that this problem exists today? Why is this important?
It hinders user experience. Widgets become too slow. Please see Doherty Threshold which states that Productivity soars when a computer and its users interact at a pace (<400ms) that ensures that neither has to wait on the other. 

### Who will benefit from it?

Power Users who use Firefox.

**Comment by cadeyrn** [comment]
Duplicate of #3597 and #13994.
